eclipse-projects/cli-test/.classpath

```diff
@@ -43,9 +43,9 @@
       <attribute name="javadoc_location" value="jar:file:lib_managed/docs/org.codehaus.woodstox/stax2-api/stax2-api-4.1-javadoc.jar!/"/>
     </attributes>
```

Comment:
The .classpath files are all changes created by the UpdateEclipseClasspath utility that maintains these files.